dashdotdb/services/deploymentvalidation.py

```diff
@@ -34,15 +36,16 @@ def _insert(self, item):
             self.log.error('skipping validation: key "value" not found')
             return
 
-        expire = datetime.now() - timedelta(minutes=50)
-        db_token = db.session.query(Token) \
-            .filter(Token.timestamp > expire).first()
+        db_token = db.session.query(Tokens) \
+            .filter(Tokens.uuid == token).first()
         if db_token is None:
-            db.session.add(Token(timestamp=datetime.now()))
-            db.session.commit()
-            self.log.info('token created')
-        db_token = db.session.query(Token) \
-            .filter(Token.timestamp > expire).first()
+            self.log.error(f'skipping validation: token not found: {token}')
+            return
+
+        if not db_token.is_open:
+            self.log.error(
+                f'skipping validation: token {token} is closed for data')
+            return
```

Comment:
what does this return to the client? I guess it should return a 4xx, but I'm unsure which one

Reply:
This will now return a 400